data/input_files/tests/solid/dyn_solve.lua

```diff
@@ -48,8 +48,7 @@ solid = {
     order = 1,
 
     -- neo-Hookean material parameters
-    mu = 0.25,
-    K  = 5.0,
+    materials = { { model = "NeoHookean", mu = 0.25, K = 5.0, density = 1.0 }, }
```

Comment:
Is `1.0` a sensible default? Can this be added to the Inlet definition call?

Reply:
do you mean setting a defaultValue when I define the schema? i think it would be fine to set it for density because its used for all of the material types, but for the others we wouldn't be able to tell whether or not the user entered the right parameters - so it would be difficult to verify

Reply:
I don't think we should have density be a user-set value. That is clearly something we need as input to make reasonable simulations. It is mostly set to 1.0 since all of our problems are currently notional examples.

Reply:
What do you mean by user-set value? An option in Inlet?

Reply:
imo, defaults should only be used if there is a natural default value, but in this case
1. a lot of different materials may be approximated by the same model, so there is no obvious "default" material choice
2. the values of model parameters for a given material depend on the user's (arbitrary) choice of units

So, I'd vote for requiring users to specify all the model parameters to avoid confusion.

Reply:
Sounds great. I was just checking due to how much of a change from the previous input files. All of these had default values and it seemed odd in retrospect.

Reply:
Sorry - I typo'd my original comment. I don't think density should have a default value.